src/components/pages/account-page/components/security-tab/components/change-password-form/ChangePasswordForm.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Box } from '@mui/material';
+import { Box, Typography, useMediaQuery } from '@mui/material';
```

Comment:
you need replace legacy components to mui

Reply:
which one? 

Box & Typografy is already imported from mui,
Input works with formik, we don't have mui formik components
Custom link - uses mui

Reply:
Input , check components -> form -> formik -> input